packages/ember-glimmer/lib/renderer.js

```diff
@@ -111,7 +135,14 @@ class Scheduler {
     if (CURRENT_TAG.validate(lastTagValue)) { return; }
     let view = this._root;
     if (view) {
-      view.renderer.rerender(view);
+      if (isEnabled('ember-glimmer-detect-two-way-reflush') ||
+          isEnabled('ember-glimmer-allow-two-way-reflush')) {
+        let shouldReflush;
+        let callback = () => view.renderer.rerender(view, { alwaysRevalidate: true });
+        do { shouldReflush = runInTransaction(callback); } while (shouldReflush);
+      } else {
+        view.renderer.rerender(view);
+      }
```

Comment:
I *think* this change is not needed/wrong, need to check my branch to be sure. What's making me suspicious is that `view.renderer.rerender` doesn't actually take the `alwaysRevalidate` option, and it already does the `runInTransaction` stuff there, so this is probably a copy/paste/merge error